tests/testthat/test-raw-data.R

```diff
@@ -0,0 +1,50 @@
+library(dplyr)
+library(omopbundles)
+library(testthat)
+library(stringr)
+library(purrr)
+
+
+
+extract_bundle_details <- function(path) {
+  filename <- stringr::str_remove(basename(path), "\\.csv$")
+
+  # get the domain
+  parts <- stringr::str_split(path, "/")[[1]]
+  domain <- parts[length(parts) - 2]
+
+  list(id = filename, domain = domain)
+}
+
+assert_bundle_has_name <- function(bundle) {
+  bundle_name_file <- omopbundles:::get_raw_dir(bundle$domain, "bundle_names.csv")
```

Comment:
```suggestion
  bundle_name_file <- get_raw_dir(bundle$domain, "bundle_names.csv")
```

Reply:
Will this work, as its not an exported function?

Reply:
Yep, the `testthat` suite loads the entire package, just like you would do with `devtools::load_all()`
I think as a general rule, `:::` should be avoided as much as possible. I see it more as an interactive tool to inspect non-exported code from other packages if I need to debug something. Pretty sure CRAN doesn't even allow you to have any `:::` at all in your package.

Reply:
Ah interesting, I must be misremembering testing using internal functions, okay great